internal/operators/cnv/cnv_operator_test.go

```diff
@@ -36,29 +37,25 @@ var _ = Describe("CNV operator", func() {
 	})
 
 	Context("getDependencies", func() {
```

Comment:
I'm not sure what are you reffering here.
This PR has 2 commits 
1. for the enablement of CNV on ARM 
2.  for issue where the default storage is LSO instead of LVM

Reply:
And the test here is testing the change that is done in the second commit. Right?